### PR TITLE
feat(dashboard): add functions to logs

### PR DESCRIPTION
**Pull request by nunopato**
image

### CODE REVIEW

File: .changeset/chatty-rabbits-learn.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/dashboard': minor
+---
+
+add functions to the log dashboard
```
Comment (elitan) [inline .changeset/chatty-rabbits-learn.md:5]:
```suggestion
Add functions to the Logs section.
```

**Review by elitan — APPROVED** [review]

**Review by elephant3 — APPROVED** [review]

File: .changeset/chatty-rabbits-learn.md (hunk 1 above)
Comment (elitan) [reply .changeset/chatty-rabbits-learn.md:5]: nvm

**Review by szilarddoro — APPROVED** [review]

**Review by guicurcio — APPROVED** [review]
Is this change dependent on any additional backend development? Or is it good to merge? @nunopato

**Comment by nunopato** [comment]
@guicurcio I am testing the backend changes now required to have this merged.

**Pull request merged**